conda_anaconda_tos/path.py

```diff
@@ -58,6 +94,9 @@
 #: OS and user specific metadata cache directory.
 CACHE_DIR: Final = Path(user_cache_dir(APP_NAME, appauthor=APP_NAME))
 
+#: Directory for backup/restore operations.
+TOS_BACKUP_DIR: Final = CACHE_DIR / "backup"
```

Comment:
I don't know what else the cache directory is used for, but since `APP_NAME` is `conda`, uninstallers currently do not capture that directory.

Maybe that's a feature request for `conda` though: that packages can specify the cache directory and/or that `conda remove --clean-caches` removes those cache locations.